edgexfoundry/app-rfid-llrp-inventory
# refactor!: Change configuration file format to YAML

BREAKING CHANGE:  Configuration file now uses YAML format, default file name is now configuration.yaml







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-rfid-llrp-inventory/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
1. start up edgex with no-secty
2. `make build`
3. run `./app-rfid-llrp-inventory -cp`
4. verify app service starts up with no error
5. stop app-rfid-llrp-inventory
6.  run `./app-rfid-llrp-inventory -cp -o`
7.  verify app service starts up with no error

## New Dependency Instructions (If applicable)
